src/components/receipt-view/data-section/DataBreakdownAggregate.js

```diff
@@ -51,12 +51,17 @@ const DataBreakdownAggregate = ({
         windowDimensions={windowDimensions}
       />
       <div className={styles.impactText}>
-        <Typography variant="subtitle1">{text[type].title}</Typography>
-        <Typography variant="body2" className={styles.body}>
-          {width && (width - min) / width >= minRatio.body
-            ? text[type].body
-            : null}
-        </Typography>
```

Comment:
Is there a way to do this with just CSS? 
With an @media on a new classname?

But also, this works and we're on the last week, I don't think it's worth changing now so we can start QA on the receipt.

Reply:
Agreed. I definitely think there are more elegant ways to do this, but given that we need it to be done I think we should leave it as it.